test/TestServerProjectsLowLevel/body-string/src/Generated/Docs/StringClient.xml

```diff
@@ -92,7 +92,7 @@ This sample shows how to call PutEmptyAsync with required request content.
 var credential = new AzureKeyCredential("<key>");
 var client = new StringClient(credential);
 
-var data = "<String>";
+var data = "";
```

Comment:
The changes in this file might be weird, but these bodies are defined as constants in their swagger with the specified values here. Therefore all these changes are expected.